README.md

```diff
@@ -82,6 +82,12 @@ Check the [Install Guide](GUIDE.md) for installation instructions.
 | CONNECTIVITY_CHECK_IMAGE | connectivity-check image to use, will update assisted-service config map with given value                                           |
 | HARDWARE_INFO_IMAGE      | hardware-info image to use, will update assisted-service config map with given value                                                |
 | INVENTORY_IMAGE          | assisted-service image to be updated in assisted-service config map with given value                                                    |
+| SERVICE_NAME             | assisted-service remote service name for nodes deployment, default: assisted-service
+| TARGET                   | assisted-service target kubernetes deployment: minikube/oc/oc-ingress, default: minikube
```

Comment:
Need explanation regarding target.

Reply:
will be used to determine whether target assisted-installer service runs on minikube or psi
similar to the TARGET from assisted-service repo
so you can do make run_full_flow_with_installation TARGET=oc and it will deploy also nodes and install them on psi

Reply:
What nodes?